api/v1beta1/neutronapi_types.go

```diff
@@ -194,6 +198,13 @@ type NeutronAPIDebug struct {
 	Service bool `json:"service"`
 }
 
+type NeutronRouteAnnotations struct {
```

Comment:
I can see us having more route annotations in the future so I made this a struct